### ISSUE TITLE
nested quoting on windows

### ISSUE THREAD

**Issue by colin-kiegel** [passcod/cargo-watch#107]
... uhm, I'm not getting it. ^^

On Linux I would do this
```bash
$cargo watch -x 'run -- query "elvis presly"'
[Running cargo run -- query "elvis presly"]
```

Or 
```bash
$cargo watch -x "run -- query \"elvis presly\""
[Running cargo run -- query "elvis presly"]
```

It's working as expected.

Now on Windows I can't get nested quotes with cargo-watch.
```posh
>cargo watch -x 'run -- query "elvis presly"'

error: Found argument 'presly' which wasn't expected, or isn't valid in this context

USAGE:
    cargo watch [FLAGS] [OPTIONS]
```

Or 

```posh
>cargo watch -x 'run -- query \"elvis presly\"'
[Running 'cargo run -- query \"elvis presly\"']
```

Note, that the inner escapes `\"` are not stripped..

Also Powershell-Escapes (and combinations) don't work

```posh
>cargo watch -s 'cargo run -- query `"elvis presly`"'

error: Found argument 'presly`' which wasn't expected, or isn't valid in this context

USAGE:
    cargo watch [FLAGS] [OPTIONS]
```

**Comment by passcod** [comment]
Hey, sorry for the late reply. I've read this, and I think I know what's going on, but I'm flat out at work coming up on the yule break and I won't have time to check this out in depth for a few more days.

**Comment by passcod** [comment]
Out of curiosity, does the latest 7.0.8 (released today) solve this?

**Comment by colin-kiegel** [comment]
cool, sounds like a useful library by itself :-)

**Comment by passcod** [comment]
Can you try again? I don't think all of your cases will have been fixed by this latest [7.1.0](https://github.com/passcod/cargo-watch/releases/tag/v7.1.0), but perhaps an improvement.
